libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -328,6 +328,23 @@ class Compiler {
     hlsl_explicit_bindings_[static_cast<int>(stage)].push_back(binding);
   }
 
+  // Sets an implicit set for the given GLSL shader if no set has been provided in
+  // the qualifier
+  void SetImplicitSet(uint32_t set)
+  {
+    for (auto stage : stages()) {
+      SetImplicitSetForStage(stage, set);
+    }
+  }
+
+  // Sets an implicit set for the given GLSL shader stage if no set has been provided in
+  // the qualifier
+  void SetImplicitSetForStage(Stage stage, uint32_t set)
+  {
+    hlsl_explicit_bindings_[static_cast<int>(stage)].clear();
```

Comment:
This interferes with the operation of SetHlslRegisterSetAndBindingForStage

Also, glslang only operates on these options in triples.
See https://github.com/KhronosGroup/glslang/blob/b2d2c9dd3d77c6adadff37309aa58a181bafc5f9/glslang/HLSL/hlslParseHelper.cpp#L6388